assets/src/dashboard/app/serializers/savedTemplates.js

```diff
@@ -22,16 +22,15 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { migrate, DATA_VERSION } from '../../../edit-story/migration/migrate';
-import { toUTCDate } from '../../../date';
```

Comment:
Do we need the function `toUTCDate` ? It is being used for anything?

Reply:
Good point, I think it is now unused.